samples/client/petstore/typescript-angular/tsd.json

```diff
@@ -5,11 +5,20 @@
   "path": "typings",
   "bundle": "typings/tsd.d.ts",
   "installed": {
-    "angularjs/angular.d.ts": {
+  	"angularjs/angular.d.ts": {
       "commit": "f6c8ca47193fb67947944a3170912672ac3e908e"
     },
     "jquery/jquery.d.ts": {
       "commit": "f6c8ca47193fb67947944a3170912672ac3e908e"
+    },
+    "request/request.d.ts": {
```

Comment:
Why do we have a dependency on request in the angular client? Isn't request only for node?